src/main/java/org/sagebionetworks/web/client/mvp/AppActivityMapper.java

```diff
@@ -110,6 +111,7 @@ public AppActivityMapper(PortalGinInjector ginjector, SynapseJSNIUtils synapseJS
 		excludeFromLastPlace.add(Trash.class);
 		excludeFromLastPlace.add(Certificate.class);
 		excludeFromLastPlace.add(SignedToken.class);
+		excludeFromLastPlace.add(SubscriptionPlace.class);

```

Comment:
remove.  test going to subscription place as anonymous, and verify that it goes back after login.